### ISSUE TITLE
Green and White Status Bar

### ISSUE THREAD

**Issue by EntouchGraphics** [peterhajas/MobileNotifier#138]
Hey I am having some issues with the status bar, after a notification is sent away for "later" the status bar turns Green for a Second and then stays a slightly transparent white until you respiring.

I know that this is being caused by Mobile Notifier because a downgrade from Beta4 to Beta3 Fixes the issue...

Just thought I would put this out there!

**Comment by peterhajas** [comment]
Thank you for dealing with this issue.

**Comment by EntouchGraphics** [comment]
I am sorry for reopening, I have found that the integrity of my iOS install was a bit buggy, so after a complete restore and re-jailbreak... Here is what I have found:

MobileNotifier Beta 4 was the first and only tweak I installed from Cydia, after a notification is sent away for "Later" it does indeed cause a momentary Green status bar.

After that I started installing some more apps that I tend to use to see what was causing the White status bar and have found it is not an issue with Mobile Notifier but with Notified using Activator for that "Android" like pull down notifications. A simple update of Notified fixed this issue.

So in Review: White Status bar is NOT an issue with mobilenotifier. A MOMENTARY GREEN STATUS BAR AFTER A NOTIFICATION IS AN ISSUE WITH MOBILENOTIFIER.

Again, I am really sorry for reopening this issue but I am just trying to help.
Good Luck and GREAT job on mobilenotifier Peter!
